build.sbt

```diff
@@ -4879,6 +4881,17 @@ lazy val cleanPolyglotRoot = taskKey[Unit](
   "Helper task that prepares polyglot directory of a stdlib component"
 )
 
+lazy val `opencv-thin` = project
+  .in(file("std-bits") / "opencv-thin")
+  .enablePlugins(JarExtractPlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.openpnp" % "opencv" % opencvVersion
+    ),
+    inputJar := "org.openpnp" % "opencv" % opencvVersion,
```

Comment:
- Why not read `libraryDependencies` directly instead of `inputJar`?
- maybe fail when there is more than one dependency (we don't need more than one right now)

Reply:
We need `java.io.File`. We need `Path` to the local file system. `libraryDependencies` does not give that to us. Unfortunately, there is no way to access library resolution of sbt, so you have to do that manually.